### PR TITLE
Update  package name for left springboot module

### PR DESCRIPTION
**Pull request by wujack778**
Update  package name for left springboot module

### CODE REVIEW

File: sdk/spring/azure-spring-boot-test-keyvault/src/test/java/com/azure/test/keyvault/ActuatorIT.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public void testSpringBootActuatorEnv() {
         }
     }
 
-    @SpringBootApplication(scanBasePackages = {"com.microsoft.azure.keyvault.spring"})
+    @SpringBootApplication(scanBasePackages = {"com.azure.spring"})
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot-test-keyvault/src/test/java/com/azure/test/keyvault/ActuatorIT.java:67]: why not `com.azure.keyvault.spring` here?

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/keyvault/spring/KeyVaultPropertySource.java [hunk 2]
```diff
@@ -1,10 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.microsoft.azure.keyvault.spring;
+package com.azure.keyvault.spring;
 
-import static com.microsoft.azure.utils.Constants.AZURE_KEYVAULT_PROPERTYSOURCE_NAME;
+import static com.azure.utils.Constants.AZURE_KEYVAULT_PROPERTYSOURCE_NAME;
 
+import com.azure.keyvault.spring.KeyVaultOperation;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/keyvault/spring/KeyVaultPropertySource.java:8]: Do we need `import com.azure.keyvault.spring.KeyVaultOperation` here?

File: sdk/spring/azure-spring-boot/src/main/resources/META-INF/spring.factories [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-com.microsoft.azure.spring.autoconfigure.gremlin.GremlinRepositoriesAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.aad.AADAuthenticationFilterAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.aad.AADOAuth2AutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.b2c.AADB2CAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.metrics.AzureMonitorMetricsExportAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.jms.ServiceBusJMSAutoConfiguration,\
-com.microsoft.azure.spring.autoconfigure.cosmosdb.CosmosHealthConfiguration,\
-com.microsoft.azure.keyvault.spring.KeyVaultHealthConfiguration
+org.springframework.boot.env.EnvironmentPostProcessor=com.azure.spring.cloudfoundry.environment.VcapProcessor,\
+com.azure.spring.KeyVaultEnvironmentPostProcessor
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/main/resources/META-INF/spring.factories:2]: the `KeyVaultEnvironmentPostProcessor` is under the `com.azure.keyvault.spring`, we have `com.azure.keyvault.spring` and `com.azure.spring` in azure-spring-boot`, don't confuse them

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbReactiveRepositoriesAutoConfigureRegistrar.java [hunk 4]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.microsoft.azure.spring.autoconfigure.cosmosdb;
+package com.azure.spring.autoconfigure.cosmosdb;
 
 import com.microsoft.azure.spring.data.cosmosdb.repository.config.EnableReactiveCosmosRepositories;
 import com.microsoft.azure.spring.data.cosmosdb.repository.config.ReactiveCosmosRepositoryConfigurationExtension;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbReactiveRepositoriesAutoConfigureRegistrar.java:7]: why these cosmos releated packages didn't update to `azure.azure`?

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbRepositoriesAutoConfiguration.java [hunk 5]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.microsoft.azure.spring.autoconfigure.cosmosdb;
+package com.azure.spring.autoconfigure.cosmosdb;
 
 import com.microsoft.azure.spring.data.cosmosdb.repository.CosmosRepository;
 import com.microsoft.azure.spring.data.cosmosdb.repository.config.CosmosRepositoryConfigurationExtension;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbRepositoriesAutoConfiguration.java:7]: same as above

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbRepositoriesAutoConfigureRegistrar.java [hunk 6]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.microsoft.azure.spring.autoconfigure.cosmosdb;
+package com.azure.spring.autoconfigure.cosmosdb;
 
 import com.microsoft.azure.spring.data.cosmosdb.repository.config.CosmosRepositoryConfigurationExtension;
 import com.microsoft.azure.spring.data.cosmosdb.repository.config.EnableCosmosRepositories;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbRepositoriesAutoConfigureRegistrar.java:7]: same as above

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/gremlin/GremlinRepositoriesAutoConfiguration.java [hunk 7]
```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.microsoft.azure.spring.autoconfigure.gremlin;
+package com.azure.spring.autoconfigure.gremlin;
 
 import com.microsoft.spring.data.gremlin.repository.GremlinRepository;
 import com.microsoft.spring.data.gremlin.repository.config.GremlinRepositoryConfigurationExtension;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/gremlin/GremlinRepositoriesAutoConfiguration.java:6]: same as above

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/gremlin/GremlinRepositoriesAutoConfigureRegistrar.java [hunk 8]
```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.microsoft.azure.spring.autoconfigure.gremlin;
+package com.azure.spring.autoconfigure.gremlin;
 
 import com.microsoft.spring.data.gremlin.repository.config.EnableGremlinRepositories;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/gremlin/GremlinRepositoriesAutoConfigureRegistrar.java:5]: same as above

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/metrics/AzureMonitorMetricsExportAutoConfiguration.java [hunk 9]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.microsoft.azure.spring.autoconfigure.metrics;
+package com.azure.spring.autoconfigure.metrics;
 
 import com.microsoft.applicationinsights.TelemetryConfiguration;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/metrics/AzureMonitorMetricsExportAutoConfiguration.java:6]: same

File: sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbRepositoriesAutoConfigurationUnitTest.java [hunk 10]
```diff
@@ -1,10 +1,10 @@
...
 
-package com.microsoft.azure.spring.autoconfigure.cosmosdb;
+package com.azure.spring.autoconfigure.cosmosdb;
 
-import com.microsoft.azure.spring.autoconfigure.cosmosdb.domain.Person;
-import com.microsoft.azure.spring.autoconfigure.cosmosdb.domain.PersonRepository;
+import com.azure.spring.autoconfigure.cosmosdb.domain.Person;
+import com.azure.spring.autoconfigure.cosmosdb.domain.PersonRepository;
 import com.microsoft.azure.spring.data.cosmosdb.core.CosmosTemplate;
 import com.microsoft.azure.spring.data.cosmosdb.repository.config.EnableCosmosRepositories;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbRepositoriesAutoConfigurationUnitTest.java:9]: same

File: sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/gremlin/GremlinAutoConfigurationUnitTest.java [hunk 11]
```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.microsoft.azure.spring.autoconfigure.gremlin;
+package com.azure.spring.autoconfigure.gremlin;
 
 import com.microsoft.spring.data.gremlin.common.GremlinFactory;
 import com.microsoft.spring.data.gremlin.conversion.MappingGremlinConverter;
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/gremlin/GremlinAutoConfigurationUnitTest.java:6]: same, same problem I won't comment again, you can check it by yourself

**Review by zhoufenqin — CHANGES_REQUESTED** [review]

**Comment by zhoufenqin** [comment]
@wujack778 I think you can run the unit test and integration test in your local env first, it helps you figure out the problem as soon as possible. you can refer to these doc:
integration-test: [here](https://dev.azure.com/azure-spring-integration/spring-integration-private/_wiki/wikis/spring-integration-private.wiki/23/Run-integration-test-and-Set-up-pipeline-for-personal-repo)
unit test: [here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/CONTRIBUTING.md#building-from-source)

**Comment by wujack778** [comment]
@zhoufenqin understood

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbReactiveRepositoriesAutoConfigureRegistrar.java (hunk 4 above)
Comment (wujack778) [reply sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbReactiveRepositoriesAutoConfigureRegistrar.java:7]: Moary will update this in his pr. Maybe need to discuss this with xiaolu.

**Comment by wujack778** [comment]
I will open a new pr for this until Moary finish the Update for cosmos modules.

**Pull request closed without merging**